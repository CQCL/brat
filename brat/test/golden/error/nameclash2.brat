type A(x) = x
type A(x,y) = y
