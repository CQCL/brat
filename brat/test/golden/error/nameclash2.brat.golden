Error in test/golden/error/nameclash2.brat@2:1-16
type A(x,y) = y
^^^^^^^^^^^^^^^

  Name clash: Identifier `A` is already used

